lib/src/clojure_lsp/queries.clj

```diff
@@ -835,3 +835,11 @@
           (filter #(and (:import %)
                         (safe-equal? class-name (:class %)))))
         (:analysis db)))
+
+(defn find-all-definitions-of-namespace [db namespace]
```

Comment:
since this function searchs only in the project and not whole analysis, can't we name it `find-all-project-namespace-definitions`?

Reply:
That's a better name, for sure.
Renamed the fn and added some more tests.